llvm/include/llvm/Analysis/MemoryLocation.h

```diff
@@ -77,6 +77,7 @@ class LocationSize {
   };
 
   uint64_t Value;
+  bool Scalable = false;
```

Comment:
This needs to be packed into Value, otherwise you're making LocationSize two times larger.